src/Sylius/Bundle/PromotionBundle/spec/Criteria/DateRangeSpec.php

```diff
@@ -43,4 +44,19 @@ function it_adds_filters_to_query_builder(DateTimeProviderInterface $calendar, Q
 
         $this->filterQueryBuilder($queryBuilder)->shouldReturn($queryBuilder);
     }
+
+    function it_verifies_catalog_promotion(
```

Comment:
We can do it with the same spec, but can we test other possibile outcomes, like null on value and returning false from verify function?